hera_cal/frf.py

```diff
@@ -1277,7 +1280,7 @@ def load_tophat_frfilter_and_write(datafile_list, baseline_list=None, calfile_li
         baseline_antennas = np.unique(baseline_antennas).astype(int)
         if calfile_list is not None:
             cals = io.HERACal(calfile_list)
-            cals.read(antenna_nums=baseline_antennas, frequencies=freqs)
+            cals.read(antenna_nums=baseline_antennas, frequencies=freqs, )
```

Comment:
I assume this is left over from something?

Reply:
yep. I've removed this.